src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,50 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ARRAY_SIZE = 10;
+    private final K[] keysArray;
+    private final V[] valuesArray;
+    private int size;
+
+    public StorageImpl() {
+        keysArray = (K[]) new Object[MAX_ARRAY_SIZE];
+        valuesArray = (V[]) new Object[MAX_ARRAY_SIZE];
+    }
+
+    private boolean isEqualsKeys(Object first, Object second) {
+        return (first == second) || (first != null && first.equals(second));
+    }
```

Comment:
let's put your private methods after public ones and also:
```suggestion
    private boolean isEqualsKeys(K first, K second) {
        return first == second || first != null && first.equals(second);
    }
```